packs/datadog/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
+api_key: ''
```

Comment:
When you get a chance, it would be great if you could also add config.schema.yaml file (support for config schema was added in a recent release of StackStorm) - https://docs.stackstorm.com/reference/pack_configs.html#configuration-schema